guides/locations/countries-and-states.md

```diff
@@ -0,0 +1,35 @@
+# Countries and states
+
+Solidus uses the list of countries and states provided by [Carmen][carmen-repo],
+which uses the countries and states available in the [Debian `iso-codes`
+package][debian-iso-codes]. Solidus creates new `Spree::Country` and
+`Spree::State` objects for each country and state.
+
+Carmen is generally up-to-date and should provide Solidus with any country or
+state you would ever need.
+
+[carmen-repo]: https://github.com/carmen-ruby/carmen
+[debian-iso-codes]: https://packages.debian.org/sid/all/iso-codes
+
+## Countries
+
+If a country or state you require is not recognized, you can add it directly
+from the Solidus admin (**Settings -> Locations -> Countries**) or from your
```

Comment:
You actually can't add states or countries from **Settings -> Locations -> ...** anymore. This needs to be updated.